backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceHeaderImplBuilder.java

```diff
@@ -121,6 +122,8 @@ public class InvoiceHeaderImplBuilder
 
 	private int auctionId;
 
+	private int orgBankAccountId;
```

Comment:
why not using BankAccountId?

Reply:
IMHO would only make sense, if it's completly refactored including `de.metas.invoicecandidate.api.impl.InvoiceHeaderImplBuilder#checkOverrideID`